<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCViewerMW</class>
 <widget class="QMainWindow" name="GCViewerMW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G-code viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="GC2DView" name="gc2DView">
           <attribute name="title">
            <string>2D</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="layerSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTreeView" name="gcTreeView">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_FileOpen"/>
    <addaction name="separator"/>
    <addaction name="action_FileQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_HelpAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_SettingsFilament"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_FileOpen"/>
  </widget>
  <action name="action_FileOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_FileQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_HelpAbout">
   <property name="icon">
    <iconset theme="help-about&#9;">
     <normaloff>../../../../../../../../../../../.designer/backup</normaloff>../../../../../../../../../../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_SettingsFilament">
   <property name="text">
    <string>Filament</string>
   </property>
   <property name="toolTip">
    <string>Filament settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GC2DView</class>
   <extends>QWidget</extends>
   <header>GC2DView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gcviewer.qrc"/>
 </resources>
 <connections/>
</ui>
